src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidationImpl.java

```diff
@@ -22,6 +27,16 @@
  */
 @Service
 public class FigaroValidationImpl implements FigaroValidation {
+	
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	
+	@Autowired
+	//DefaultApi defaultApi;
+	FigvalidationsApi ordsapi;  
+	
+    public FigaroValidationImpl(FigvalidationsApi ordsapi) {
```

Comment:
If you have @autowired, I  don't think you need to use the constructor injection, I usually use constructor injection over @autowired, so I don't need the spring boot context to run the unit tests.

Reply:
Correct. It was left over from some earlier development. I've removed. Thx